inputFiles/hydraulicFracturing/heterogeneousInSitu_base.xml

```diff
@@ -172,7 +175,7 @@
     <FiniteVolume>
       <TwoPointFluxApproximation
         name="singlePhaseTPFA"
-        meanPermCoefficient="1.0"/>
+        meanPermCoefficient="0.0"/>
```

Comment:
yes, this is the key change to make it run

Reply:
I have tested this example with @zhishuaizhang in the workshop. I think we can keep the harmonic average, i.e., `meanPermCoefficient="1.0"`, and a better solution I believe is to set a nonzero `penaltyStiffness` (e.g., 1e12) as we have negative `elementAperture` here. We may also need more Newton Iteration allowance, but it can converge now with `penaltyStiffness=1e12`.

Reply:
oh ok, interesting, thanks
i didn't try to change the penalty coefficient, can you share the output of that run?

Reply:
> oh ok, interesting, thanks
> i didn't try to change the penalty coefficient, can you share the output of that run?

here it is 
[log.txt](https://github.com/GEOS-DEV/GEOS/files/13168248/log.txt)

Reply:
> > oh ok, interesting, thanks
> > i didn't try to change the penalty coefficient, can you share the output of that run?
> 
> here it is [log.txt](https://github.com/GEOS-DEV/GEOS/files/13168248/log.txt)

And here are the blocks I've changed. I've also changed the `NonlinearSolverParameters` to make sure it can converge. 

`<FrictionlessContact
      name="fractureContact"
      penaltyStiffness="1e12"
      apertureTableName="apertureTable"/>`

`<FiniteVolume>
      <TwoPointFluxApproximation
        name="singlePhaseTPFA"
        meanPermCoefficient="1.0"/>
    </FiniteVolume>`

`<Hydrofracture
      name="hydrofracture"
      solidSolverName="lagsolve"
      flowSolverName="SinglePhaseFlow"
      surfaceGeneratorName="SurfaceGen"
      logLevel="1"
      targetRegions="{ Fracture }"
      contactRelationName="fractureContact"
      maxNumResolves="2">
      <NonlinearSolverParameters
        newtonTol="1.0e-4"
        newtonMaxIter="40"
        lineSearchMaxCuts="3"/>
      <LinearSolverParameters
        solverType="gmres"
        preconditionerType="mgr"/>
    </Hydrofracture>`

Reply:
should this be put back to 1.0?

Reply:
yes, let me do that

Reply:
@rrsettgast should be good to merge now

Reply:
I'm looking at the result of the modeling result after the fix and it seems to be different from the modeling result from the original documentation. Do with need to figure out why there is a difference? Or it is for another pull request?
![image](https://github.com/GEOS-DEV/GEOS/assets/4632742/21496f07-f119-4e9c-950e-8157f4cf76fe)

Reply:
@zhishuaizhang please post results with meanPermCoefficient=1